tests/VersionTest.php

```diff
@@ -125,6 +129,16 @@ public function test_it_can_be_greater_than_another_semver_object()
         $this->assertTrue($this->version->gt(new SemVer\Version('v1.3.36')));
         $this->assertFalse($this->version->gt(new SemVer\Version('v1.3.38')));
         $this->assertFalse($this->version->gt(new SemVer\Version('v1.3.37')));
+        $this->assertTrue($this->version->gt(new SemVer\Version('v1.3.35')));
```

Comment:
Is this new line necessary? Isn't this case already covered by line 129 above?

Reply:
You're right. It's good enough and I will remove this additional test case.